.github/config.yml

```diff
@@ -19,6 +19,7 @@ firstPRMergeComment: |
  Congrats on merging your first pull request! 🙌🎉⚡️ 
  Your code will likely be published to http://infragram.org in the next few days. 
  In the meantime, can you tell us your Twitter handle so we can thank you properly?
+ yes offcourse my Twitter Handle is @iMohsin_ali
```

Comment:
Delete both of these lines, commit and push again.

Could you please confirm this is correct? @TildaDares

Reply:
Yes that is correct @PeculiarE. Thank you!